flink-connectors/flink-connector-aws-base/src/main/java/org/apache/flink/connector/aws/util/AWSGeneralUtil.java

```diff
@@ -274,6 +275,7 @@ public static SdkAsyncHttpClient createAsyncHttpClient(
     public static SdkAsyncHttpClient createAsyncHttpClient(
             final AttributeMap config, final NettyNioAsyncHttpClient.Builder httpClientBuilder) {
         httpClientBuilder
+                .eventLoopGroupBuilder(SdkEventLoopGroup.builder())
```

Comment:
Looks like the httpClient closes the ELG when it itself is closed: https://github.com/aws/aws-sdk-java-v2/blob/master/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java#:~:text=public%20void%20close()%20%7B

Reply:
Despite this in the [JavaDocs](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.Builder.html#:~:text=The%20EventLoopGroup%20MUST%20be%20closed%20by%20the%20caller%20when%20it%20is%20ready%20to%20be%20disposed.)

Reply:
Thanks for adding a test to verify this too 👍